### PR TITLE
SIDM-3692 Increment version and disable details.

### PR DESCRIPTION
**Pull request by dfourn**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/SIDM-3692

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/idam/health/indicator/HealthCheck.java [hunk 1]
```diff
@@ -28,14 +28,15 @@ public Health health() {
...
+                String details = indicator.getDetails();
+                if (details != null) {
+                    builder.withDetail(indicator.getClass().getName(), details);
+                }});
         }
         return builder
-                .withDetail("v", defaultIfEmpty(getClass().getPackage().getImplementationVersion(), "dev"))
+                //TODO Put the version in the jar manifest.
+                .withDetail("v", defaultIfEmpty(getClass().getPackage().getImplementationVersion(), "2.0.3"))
                 .build();
```
Comment (nikola-naydenov-hmcts) [inline src/main/java/uk/gov/hmcts/reform/idam/health/indicator/HealthCheck.java]: image

Comment (dfourn) [reply src/main/java/uk/gov/hmcts/reform/idam/health/indicator/HealthCheck.java]: I've seen this but it feels like a workaround. I think it is better to put the right version in the MANIFEST.

File: build.gradle [hunk 2]
```diff
@@ -19,6 +19,12 @@ repositories {
     jcenter()
 }
 
+bootJar {
```
Comment (radoslaw-orlowski-amido) [inline build.gradle]: Doesn't `bootJar` inherit from `jar` anyway?

Comment (dfourn) [reply build.gradle]: It didn't work for me at the time

Comment (dfourn) [reply build.gradle]: I've tested again with the new gradle and doesn't work either. Has to be bootJar for some magical reason.

**Pull request merged**